client/js/test/table-view.test.js

```diff
@@ -10,6 +10,36 @@ describe('table-view', () => {
         document.documentElement.innerHTML = html;
```

Comment:
Same thing here. What are edge cases and non-happy paths that you can anticipate a user getting into, that you can test for here?